docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 version: '2.1'
 
 services:
-  server:
+  balena-device-ui:
```

Comment:
When I was looking at the names of the containers in the Balena Dashboard, it being called 'server' was a little odd. Changed it back to this for now, perhaps you will come up with something better in the future.